host/crypto/bcrypt/ec.c

```diff
@@ -96,8 +96,8 @@ static oe_result_t _fixup_public_key_info_for_import(
          * offsets from the original */
         new_key_info = (PCERT_PUBLIC_KEY_INFO)new_key_info_buffer;
         new_key_info->Algorithm.pszObjId =
-            new_key_info_buffer +
-            (key_info->Algorithm.pszObjId - key_info_base);
+            (LPSTR)new_key_info_buffer +
```

Comment:
I believe the LP types are obsolete and these should just be (PSTR).   (Longer explanation: as noted in https://stackoverflow.com/questions/2315311/what-is-a-long-pointer, the LP types are a holdover from Win16 days, and OE won't run on Win16 anyway, so we should use current types instead of obsolete ones.)